test/screenshotter/ss_data.yaml

```diff
@@ -253,6 +253,7 @@ MathNormal: \mathnormal{Ax2k\breve{a}\omega\Omega\imath+\KaTeX}
 MathOp: a\mathop+b\mathop:c\mathop{\delta}e\mathop{\textrm{and}}f\mathrel{\mathop{:}}=g\sin h
 MathRm: \mathrm{Ax2k\breve{a}\omega\Omega\imath+\KaTeX}
 MathSf: \mathsf{Ax2k\breve{a}\omega\Omega\imath+\KaTeX}
+MathSfit: \mathsfit{Ax2k\breve{a}\omega\Omega\imath+\KaTeX}
```

Comment:
This screenshot test is good, but needs corresponding image files. It looks like there's a bug in the CI that this isn't erroring or outputting the necessary files. I'll investigate.

Reply:
Renamed

Reply:
I was able to generate the screenshots by hand (running `yarn test:screenshots:update` with a browserstack key), so we're all set here!